index.html

```diff
@@ -7040,6 +7062,18 @@ <h4 id=input-source-state>State</h4>
  a <a>pointer input state</a> object with <code>subtype</code> set
  to <var>subtype</var>, <code>pressed</code> set to an empty set and
  both <code>x</code> and <code>y</code> set to <code>0</code>.
+	
+<p>A <a>wheel input source</a>’s <a>input source state</a> is a
```

Comment:
I don't think this, or the next paragraph are actually used? I would expect a scrollwheel to have its own state, but I'd expect it to share state with the pointer (arguably for that reason it could be `pointerScrollWheel`, but I'm not sure).